src/components/Form/Login.tsx

```diff
@@ -69,7 +96,21 @@ const Login = () => {
             <Link to="/forgot-password">Fogot password?</Link>
           </Form.Text>
         </Form.Group>
-        <Button type="submit" variant="primary" className={styles.formSubmit}>
+        <Button
+          type="submit"
+          variant="primary"
+          className={styles.formSubmit}
+          disabled={mutation.isLoading}
+        >
+          {mutation.isLoading && (
+            <Spinner
+              as="span"
+              animation="border"
+              size="sm"
+              role="status"
+              className="me-2"
```

Comment:
Could you please extract the Spinner component and use it in the Loader component? We can pass `...props` to the Loader component. I will use it in my tasks.